api/swagger-spec/oapi-v1.json

```diff
@@ -13819,6 +13819,11 @@
      "resources": {
       "$ref": "v1.ResourceRequirements",
       "description": "the desired compute resources the build should have"
+     },
+     "maxDuration": {
```

Comment:
We've talked with @bparees about naming and decided to go with `maxDuration` as it speaks more clearly what the field does than `activeDeadlineSecods`. 
Other than that if sometime in the future we'll decide to allow multi-pod builds, we'd still have `maxDuration` which will leverage job's `maxDuration` (or whatever it'll be called).